### DISCUSSION TITLE
colors---> module not showing

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14490] [General] [answered]
i cannot see any modules here ? do i need to enable any setting ? or is it under development ?

![image](https://user-images.githubusercontent.com/54880467/113344598-e72ab500-934e-11eb-96fb-14412e2d00cb.png)

**Comment by vovpff** [comment]
I think this feature not finished yet and unavailable for users

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
Fixed https://github.com/YetiForceCompany/YetiForceCRM/commit/ef0e76a6765e0b5bcf51830b39d7a2fd90bbf97e
